.gitignore

```diff
@@ -40,4 +40,7 @@ jspm_packages
 # Optional REPL history
 .node_repl_history
 
+# IDE files
```

Comment:
I would propose backing out personal editor ignores based on this philosophy https://jaketrent.com/post/gitignore-system-files/